dubbo-config/dubbo-config-api/pom.xml

```diff
@@ -68,6 +68,11 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-rpc-triple</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
```

Comment:
What is the purpose of adding this dependency? Can it be changed to the test scope?